Plugins/Wox.Plugin.Folder/Main.cs

```diff
@@ -191,9 +191,14 @@ private List<Result> QueryInternal_Directory_Exists(Query query)
             if (incompleteName.StartsWith(">"))
             {
                 searchOption = SearchOption.AllDirectories;
-                incompleteName = incompleteName.Substring(1);
+
+                // match everything before and after search term using supported wildcard '*', ie. *searchterm*
+                incompleteName = "*" + incompleteName.Substring(1);
```

Comment:
I think this was my initial implementation on this and it made sense to me but while checking it had some trouble but to be honest I don't remember what was the issue with adding it.

Reply:
Seems to work when I test it. The package in the prerelease contains this change, you can download and test it out too.